test/mongodb.test.js

```diff
@@ -88,7 +88,7 @@ describe('mongodb', function () {
       });
       ds.ping(function(err) {
         (!!err).should.be.true;
-        err.message.should.be.equal('failed to connect to [localhost:4]');
+        err.message.should.be.equal('connect ECONNREFUSED');
```

Comment:
Please check that the "Test Connection" button converts this new error message into a useful message in the GUI, so that we don't introduce a subtle regression there.